redfish-core/lib/log_services.hpp

```diff
@@ -912,7 +959,16 @@ inline void createDumpTaskCallback(
             std::string prop;
             m.read(prop, values);
 
-            if (getDumpCompletionStatus(values))
+            int ret = getDumpCompletionStatus(values, taskData, createdObjPath);
```

Comment:
status? dumpStatus? 
Not really a fan of "ret" here, it doesn't tell you much

Reply:
Done